src/apps/explorer/pages/Metadata/config.ts

```diff
@@ -0,0 +1,214 @@
+import { AjvError, FormValidation, UiSchema } from '@rjsf/core'
+import {
+  LATEST_APP_DATA_VERSION,
+  LATEST_QUOTE_METADATA_VERSION,
+  LATEST_REFERRER_METADATA_VERSION,
+} from '@cowprotocol/app-data'
+import { JSONSchema7 } from 'json-schema'
+
+const ERROR_MESSAGES = {
+  REQUIRED: 'Required field.',
+  INVALID_ADDRESS: 'This is not an address.',
+  ONLY_DIGITS: 'Only digits are allowed.',
+}
+
+export const INITIAL_FORM_VALUES = {
+  version: LATEST_APP_DATA_VERSION,
+  metadata: {
+    referrer: {},
+    quote: {},
+  },
+}
+
+export const getSchema = async (): Promise<JSONSchema7> => {
+  const { default: latestSchema } = await import(`@cowprotocol/app-data/schemas/v${LATEST_APP_DATA_VERSION}.json`)
+  return formatSchema(latestSchema)
+}
+
+const formatSchema = (schema: JSONSchema7): JSONSchema7 => {
+  const formattedSchema = structuredClone(schema)
+
+  deletePropertyPath(formattedSchema, 'properties.appCode.examples')
+  deletePropertyPath(formattedSchema, 'properties.version.examples')
+  deletePropertyPath(formattedSchema, 'properties.environment.examples')
+  deletePropertyPath(formattedSchema, 'properties.metadata.properties.referrer.properties.address.examples')
+  deletePropertyPath(formattedSchema, 'properties.metadata.properties.quote.properties.slippageBips.examples')
```

Comment:
Since you are already navigating through the schema, what about writing a method to remove a property by name:

```js
deleteAllPropertiesByName(schema, 'examples')
```